php_sql/task1/index.php

```diff
@@ -0,0 +1,73 @@
+<?php 
+    $valid=array();
+    $error=array();
+    $maxSizeCourse =150;
+    $maxSizeName=200;
+    $minLengthDescription=10;
+    $numberFileds=3;
+    function validate ($name, $limit, &$valid, &$errors)
+    {
+        $check=$_POST["$name"];
+        if (!$check)
+        {
+            $errors["$name"]="$name е задължително поле";
+        }
+        elseif(strlen("$name")>$limit)
+        {
+            $errors["$name"]="$name трябва да бъде не повече от $limit символа";
+        }
+        else
+        {
+            $valid["$name"]=$check;
+        }
+    }
+    function check ($name, &$valid, &$errors)
+    {
+        $var=$_POST["$name"];
+        if (!$var)
+        {
+            $errors["$name"]='Полето е задължително';
+        }
+        else
+        {
+            $valid["$name"]=$var;
+        }
+    }
+    if ($_POST) 
+    {
+        validate("title", $maxSizeCourse, $valid, $errors);
+        validate("name", $maxSizeName, $valid, $errors);
+        $description=$_POST['description'];
+        if (!$description)
```

Comment:
I like that you use validation function to validate the fields. You can use it to validate description field, too.